openshift/prerequisites.rst

```diff
@@ -7,7 +7,7 @@ Prerequisites for GPU Operator on OpenShift
 
 Before following the steps in this guide, ensure that your environment has:
 
-* A working OpenShift cluster up and running with a GPU worker node. See `OpenShift Container Platform installation <https://docs.openshift.com/container-platform/latest/installing/index.html>`_  for guidance on installing.
+* A working OpenShift cluster up and running with a GPU worker node. See `OpenShift Container Platform installation <https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/installation_overview/ocp-installation-overview>`_  for guidance on installing.
```

Comment:
```suggestion
* A working OpenShift cluster up and running with a GPU worker node. See `OpenShift Container Platform installation <https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/installation_overview/ocp-installation-overview>`_  for guidance on installing.
```

Let's use `latest` instead. I've verified that it works and I presume it automatically redirects to the latest available major.minor